test/MUXExperimentalTest/MUXExperimentalTestApp/MUXExperimentalTestApp.csproj

```diff
@@ -48,7 +48,7 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
-    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```

Comment:
Why these changes?

Reply:
For some reason, BinSkim was choking on this test app. It looks like it was not able to handle apps built with this enabled. Since there is no way in the template to exclude this exe from scanning, this was the easiest way to resolve the issue.